rs/nervous_system/common/src/lib.rs

```diff
@@ -76,6 +76,10 @@ pub const SNS_CREATION_FEE: u64 = 180 * ONE_TRILLION;
 // The number of nanoseconds per second.
 pub const NANO_SECONDS_PER_SECOND: u64 = 1_000_000_000;
 
+/// Maximum allowed number of SNS neurons for direct swap participants that an SNS may create.
+/// This constant must not exceed `NervousSystemParameters::MAX_NUMBER_OF_NEURONS_CEILING`.
+pub const MAX_NEURONS_FOR_DIRECT_PARTICIPANTS: u64 = 100_000;
```

Comment:
This constant needed to be moved as Swap does not have sns-init as a dependency.